platform/mellanox/fw.mk

```diff
@@ -52,7 +52,12 @@ MLNX_SPC4_FW_FILE = fw-SPC4-rel-$(subst .,_,$(MLNX_SPC4_FW_VERSION))-EVB.mfa
 $(MLNX_SPC4_FW_FILE)_PATH = $(MLNX_FW_BASE_PATH)
 $(MLNX_SPC4_FW_FILE)_URL = $(MLNX_FW_BASE_URL)/$(MLNX_SPC4_FW_FILE)
 
-MLNX_FW_FILES = $(MLNX_SPC_FW_FILE) $(MLNX_SPC2_FW_FILE) $(MLNX_SPC3_FW_FILE) $(MLNX_SPC4_FW_FILE)
+MLNX_SPC5_FW_VERSION = 37.2014.3050
```

Comment:
Current upstream FW (for 202412) is: 2014.3094
But, it does not support SPC5. Need to be aligned.